src/nxt_otel.c

```diff
@@ -97,6 +97,29 @@ nxt_otel_span_add_headers(nxt_task_t *t, nxt_http_request_t *r)
         }
     } nxt_list_loop;
 
+    // Add method and path to the trace as well
+    // 1. method first
+    name_cur = val_cur = NULL;
+    name_cur = nxt_mp_zalloc(r->mem_pool, 7); // the word "method" is 6 chars + 1 for nullterm
+    val_cur = nxt_mp_zalloc(r->mem_pool, r->method->length + 1);
+    if (name_cur && val_cur) {
+        strncpy((char *) name_cur, (char *) "method", 7);
```

Comment:
I want "method" to be a macro just like NXT_OTEL_BODY_SIZE_TAG at the top of this file.
I want 7 to be more like sizeof(MACRO_NAME) so that we dont have to update them seperately.

Reply:
Fixed this in a force push